tests/features/external_exposure_test.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from conftest import FakeRandom
+
+
+@pytest.mark.unit
+def test_external_exposure(make_model, make_agent, params):
+    params.features.external_exposure = True
+
+    model = make_model(params)
+    external_exposure_agent = make_agent()
+    external_exposure_agent.external_exposure.active = True
+    model.pop.add_agent(external_exposure_agent)
+
+    model.run_random = FakeRandom(-0.1)
+
+    external_exposure_agent.external_exposure.update_agent(model)
+    assert external_exposure_agent.hiv.active
```

Comment:
Does the agent always initialize with `hiv.active = False`? I'm not sure, but if it's not sure, should set it that way.

Looks like there might be a missing newline at the end of this file.

Reply:
yeah, active always starts false on agent creation

Reply:
👍 I thought so but wasn't sure if the conftest agent creation altered it or something. Which wouldn't make sense, in retrospect